### DISCUSSION TITLE
Show soketi version on log

### DISCUSSION THREAD

**Discussion by angeljqv** [soketi/pws#601] [Q&A] [answered]
Is possible also showing soketi version on logger start?
https://github.com/soketi/soketi/blob/c9da87ab59d1c4eb1ecf4803ea17045932cee4ad/src/server.ts#L384-L387
Like 
```ts
Log.successTitle(`⏱ Soketi Version is ${this.version}!`); 
```

**Comment by parallels999** [comment, marked as answer]
- #602

**Reply by parallels999** [reply]
- https://github.com/soketi/soketi/pull/627
